packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -77,13 +77,23 @@ export function isNavigationCandidate( element, keyCode, hasModifier ) {
  * Renders focus capturing areas to redirect focus to the selected block if not
  * in Navigation mode.
  */
-function FocusCapture( { selectedClientId, isReverse, containerRef } ) {
+const FocusCapture = forwardRef( ( {
```

Comment:
Things are becoming a little bit hard to follow for me. it would be good to clarify exactly what this component is for, its props...

Reply:
Sorry, I should be documenting better! I'll write something for it.

Reply:
Added some in #19314.